pkg/trace/traceutil/normalize.go

```diff
@@ -113,11 +115,20 @@ func fallbackService(lang string) string {
 
 const maxTagLength = 200
 
-// NormalizeTag applies some normalization to ensure the tags match the backend requirements.
+// NormalizeTag applies some normalization to ensure the full tag_key:tag_value matches the backend requirements.
 func NormalizeTag(v string) string {
+	return normalize(v, true)
+}
+
+// NormalizeTagValue applies some normalization to ensure the tag values match the backend requirements.
```

Comment:
Can you add some more details to this comment here? What makes this function different from `NormalizeTag` etc?